inc/class.plugin.php

```diff
@@ -151,6 +151,23 @@ public static function update_option() {
 			self::load_option();
 		}
 
+		// In case fields value < 0 or null override the value of fields.
+		if (self::$options['auto_link_min'] < 0 || self::$options['auto_link_min'] === '') {
+			self::$options['auto_link_min'] = 1;
```

Comment:
Hi @anhredweb, set the <input type="number" /> minimum value to zero instead as that should prevent submission when negative value is entered instead of allowing user to submit before updating to default.

Reply:
@stevejburge I see, the bess way is add min=0 is the best way and that help to prevent from user UI, but there are 02 reason that why I did it:
1. As I debug it, the DOM elements generate through  `wp-content/plugins/simple-tags/inc/helper.options.admin.php`. So as the result not found the support from wp that can help add the attribute `min` into as the solution we are looking for.
![image](https://user-images.githubusercontent.com/6948022/126956265-5d95ac57-728b-46c0-8cfd-b707f8b5f135.png)

2. For security reason, the prevention at server side is better the only prevention only from UI which are maybe change by `inspects` option by browser then summit. That maybe lead to un-predicted result to the system.

Could you please check and feedback to me.
Thanks.

Reply:
Hi @anhredweb, every code that output the elements on TaxoPress pages belongs to TaxoPress and can be found in the plugin source code including the number text input.

Ofcourse, it's a good idea to include a server side validation and in that case, it'll still be better to output error message to user rather than just updating default for them.

But that aside, the primary request for this task is to prevent user submission if negative value is entered which can be done using javascript way or adding another option to accept min and max for the text input.

For example, you can add another option to the screenshot shared above like this:

```
array(
            'rp_limit_qty',
            __('Maximum number of related posts to display: (default: 5)', 'simpletags'),
            'number',
            'regular-text',
            '',
            'legacy-tab-content legacy-related-posts-content st-hide-content',
            **'0',// THIS IS THE MIN value** 
        ),
```

So, on the rendering page, you can check if $array[6] is set and if so, attach min="$array[6]" which is the min position

Reply:
Hi @stevejburge 

Thanks for your feedback and this help much. I'll add up min value.